src/advanced/bitcoin-anchoring.yaml

```diff
@@ -0,0 +1,259 @@
+        along with the hash of the Exonum transaction published in this LECT.
+
+        If the specified validator `id` is greater or equal to the total validators amount
+        then an error is returned.
+      parameters:
+        - in: query
+          name: id
+          required: true
+          description: Identifier of the validator to query for.
+          schema:
+            type: integer
+            minimum: 0
+            maximum: 4294967295
+      responses:
+        200:
+          description: >
+            JSON object descibing actual LECT as viewed by the specified validator.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  hash:
+                    allOf:
+                      - $ref: "#/components/schemas/Hash"
+                      - description: >
+                          Hash of Exonum transaction
+                          where the specified validator published this LECT.
+                  content:
+                    $ref: "#/components/schemas/AnchoringInfo"
+                example:
+                  hash: "c1b20563e3db4041bfb30da589b6f25a22bb19d02ed8c81abf32461f0634b784"
+                  content:
+                    txid: "021dd89bd3343a8a6ad259fbe1eed638217358b262db66a9619af2ca92fb89d9"
+                    payload:
+                      block_hash: "03c5d221357d5d10c20792d480ba29267f3895575fbe36bef175abab9e9c9f5a"
+                      block_height: 0
+                      prev_tx_chain: null
+        400:
+          description: >
+            Invalid validator `id`.
+          content:
+            application/json:
+              schema:
+                type: string
+                example: "Unknown validator id=100"
+
+  /nearest_lect:
+    get:
+      tags: [Read Requests]
+      summary: Get nearest LECT for a block
+      description: >
+        Returns the content of the anchoring transaction which anchors the specific block.
+
+        Returns `null` on any of the following conditions:
+
+        - The selected block has not been anchored yet.
+
+        - The [observer interval](https://exonum.com/doc/advanced/bitcoin-anchoring/#observer-interval)
+          is not set.
+
+        - The number of confirmations of anchoring transaction is less than
+          the [UTXO confirmations](https://github.com/exonum/exonum-btc-anchoring/blob/master/DEPLOY.md#change-configuration-parameters)
+          parameter.
+      parameters:
+        - in: query
+          name: height
+          required: true
+          schema:
+            $ref: "#/components/schemas/Height"
+      responses:
+        200:
+          description: >
+            A string with hex-encoded content of the nearest Bitcoin anchoring transaction.
+          content:
+            application/json:
+              schema:
+                type: string
+                nullable: true
+                format: "^[0-9a-f]+$"
```

Comment:
Are you sure this works? I was under the impression (reading the OpenAPI spec) that [ECMA-262 regexps](https://www.ecma-international.org/ecma-262/5.1/#sec-7.8.5) should be used (i.e., ones used in JavaScript; something like `/^[0-9a-f]+$/i`). Additionally, shouldn't the property be `pattern`, rather than `format`?